api/valor_api/backend/models.py

```diff
@@ -99,7 +100,8 @@ class Prediction(Base):
         ForeignKey("annotation.id"), nullable=True
     )
     label_id: Mapped[int] = mapped_column(
-        ForeignKey("label.id"), nullable=False
+        ForeignKey("label.id"),
+        nullable=True,
```

Comment:
This change also confuses me as the Prediction table exists primarily to link labels. Is text generation storing scores here?

Reply:
See my comment for groundtruths